libraries/platforms/bigblade-verilator/bsg_manycore_regression_platform.c

```diff
@@ -4,31 +4,28 @@
 #include <dlfcn.h>
 
 // This function is main for Verilator cosimulation
-void main(int argc, char **argv) {
+int main(int argc, char **argv) {
 
         // To reduce the number of Verilator compilations we do, this
         // platform compiles the executable machine and then loads the
         // program as a shared object file. This shared object is
         // passed as the string sopath and _must_ define a method
-        // verilator_main that can be called as the main function of
+        // sim_main that can be called as the main function of
         // the program
 
-        // We use VCS main, so that we don't have to define different
```

Comment:
making this change now since we're updating the interface

Reply:
We delayed this change since some applications in the flow still have hardcoded `vcs_main` call